test/E2E/e2e-basic.mlir

```diff
@@ -9,3 +9,21 @@ func @rank1(%arg0: tensor<?xf32>, %arg1: tensor<?xf32>) -> tensor<?xf32> {
   %0 = "tcf.add"(%arg0, %arg1) : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
   return %0 : tensor<?xf32>
 }
+
+// -----
+// CHECK-LABEL: func @multiple_ops
+func @multiple_ops(%arg0: tensor<f32>, %arg1: tensor<?xf32>, %arg2: tensor<?x?xf32>) -> tensor<?x?xf32> {
+  %0 = "tcf.add"(%arg1, %arg2) : (tensor<?xf32>, tensor<?x?xf32>) -> tensor<?x?xf32>
+  %1 = "tcf.add"(%arg0, %0) : (tensor<f32>, tensor<?x?xf32>) -> tensor<?x?xf32>
+  return %1 : tensor<?x?xf32>
+}
+
+// -----
+// CHECK-LABEL: func @global_add
+func @global_add() -> tensor<2xf32> attributes {iree.module.export} {
```

Comment:
can you put this one in a separate file e2e-constants.mlir? The failure mode for this one is somewhat different from the others in this file, so scanning lit failures based on file names saves some time diagnosing failures.